src/lib/converter/converter.ts

```diff
@@ -19,6 +19,12 @@ import { IMinimatch } from "minimatch";
 import { hasAllFlags, hasAnyFlag } from "../utils/enum";
 import { resolveAliasedSymbol } from "./utils/symbols";
 
+export interface PackageProgram {
```

Comment:
I'm not that happy with the name `PackageProgram` for this concept, any suggestions?

Reply:
`DocumentationEntrypoint` perhaps? This is a hard thing to name...